doc/source/reference/random/multithreading.rst

```diff
@@ -19,24 +19,26 @@ seed will produce the same outputs.
 
 .. code-block:: ipython
 
-    from numpy.random import Generator, PCG64
```

Comment:
Leave as `Generator` - reduces the number of changes.

Reply:
I've changed it as requested. It's a bit confusing for me though that the documentation of `Generator` has the  `See also` section saying:

 ```
See also

default_rng
    Recommended constructor for Generator.
```

Reply:
would changing Recommended -> Convenience help?

Reply:
Maybe something like:

```
See also

default_rng
    Convenience constructor for Generator with PCG64 BitGenerator.
```